pyproject.toml

```
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "6.1.1"
+version = "7.0.0-beta.1"
```

I think the [normal version](https://packaging.python.org/en/latest/specifications/version-specifiers/#pre-releases) would be:

```suggestion
version = "7.0.0beta1"
```

Or even just `7.0beta1`